### ISSUE TITLE
TDE breaks `alicloud_kvstore_instance` resource

### ISSUE THREAD

**Issue by martinkubrak** [aliyun/terraform-provider-alicloud#5977]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
```
Terraform: v1.3.6
Alicloud provider: v1.200.0
```

### Affected Resource(s)
- alicloud_kvstore_instance

### Terraform Configuration Files
```hcl
resource "alicloud_kvstore_instance" "this" {
  db_instance_name  = "${var.name}-${var.environment}-tf"
  instance_class    = var.instance_class
  instance_type     = var.instance_type
  engine_version    = var.engine_version
  zone_id           = var.zone_id
  secondary_zone_id = var.secondary_zone_id
  vswitch_id        = var.vswitch_id
  security_ips      = var.security_ips
  vpc_auth_mode     = var.vpc_auth_mode
  password          = var.password
  payment_type      = var.payment_type
  period            = var.period
  auto_renew        = var.auto_renew
  auto_renew_period = var.auto_renew_period
  config            = var.config
  backup_period     = var.backup_period
  backup_time       = var.backup_time
  tde_status        = "Disabled"
  tags              = var.tags
}

```

### Panic Output
```
│ Error: [ERROR] terraform-provider-alicloud/alicloud/resource_alicloud_kvstore_instance.go:683:
│ [ERROR] terraform-provider-alicloud/alicloud/service_alicloud_r_kvstore.go:650: Resource r-xxxxxxx DescribeInstanceTDEStatus Failed!!! [SDK alibaba-cloud-sdk-go ERROR]:
│ SDKError:
│    StatusCode: 400
│    Code: InstanceType.NotSupport
│    Message: code: 400, Current instance type does not support this operation. request id: xxxxxx-xxx-xxxx-A81F-D0D435A9DCD3
│    Data: {"Code":"InstanceType.NotSupport","HostId":"r-kvstore.aliyuncs.com","Message":"Current instance type does not support this operation.","Recommend":"https://next.api.aliyun.com/troubleshoot?q=InstanceType.NotSupport\u0026product=R-kvstore","RequestId":"xxxxxxx-3BC1-5F1B-A81F-D0D435A9DCD3"}
```

### Expected Behavior
No panic

### Actual Behavior
Terraform is unable to run plan.

### Steps to Reproduce
1. `terraform plan`

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- [GH-1234](https://github.com/aliyun/terraform-provider-alicloud/issues/2646)

**Comment by Zokiio** [comment]
We can see the same issue when trying to upgrade to v1.200.0

**Comment by martinkubrak** [comment]
Seems to be fixed in 1.201

**Comment by jerry303227** [comment]
This issue still happening in alicloud_redis_tair_instance and alicloud_kvstore_account
